draft-cookies-having-independent-partitioned-state.md

```diff
@@ -10,24 +10,25 @@ date:
 consensus: true
 v: 3
 # area: AREA
-# workgroup: WG Working Group
+workgroup: PrivacyCG
```

Comment:
Hmm isn't that HTTP WG at this point? 🤔

Reply:
Talked about this with @jyasskin and as we're not 100% sure what the correct entry for this field would be at the moment, you may just be able to omit it?

However, the draft name should probably include "httpbis" even though we're incubating in Privacy CG to make it clear what the intended standardization venue is, see https://authors.ietf.org/naming-your-internet-draft#group (also because it seems a little complicated to change draft names retroactively).

Reply:
Done.